admin/src/components/Login.js

```diff
@@ -0,0 +1,48 @@
+import "bootstrap/dist/css/bootstrap.min.css";
+import React, { useState } from "react";
+import Form from "react-bootstrap/Form";
+import Button from "react-bootstrap/Button";
+import "./login.css";
+
+const LoginPage = () => {
+  const [email, setEmail] = useState("");
+
+  const axios = require("axios");
+
+  async function getMail(email) {
+    try {
+      await axios.post("/v1/api/login", {
+        emailAddress: email,
+      });
+    } catch (error) {
+      console.error("Hiba történt a kérés során!");
+    }
+  }
+
+  return (
+    <Form
+      onSubmit={(e) => {
+        getMail(email);
+        alert(
+          "E-mail küldése megtörtént! A csatolt linkre kattintva bejelentkezhet!"
+        );
```

Comment:
Szebb lenne suttyó böngészős alert helyett a Küldés gomb alatt egy üzenet HTML-ből.

Reply:
A másik, hogy ha elhal az API kérés akkor ne mondjuk azt a felhasználónak, hogy kiment a levél.

Reply:
A harmadik, hogy a getMail async, tehát még az API kérés sem fejeződött be, mire már azt mondjuk, hogy megtörtént.